org.compiere.db.oracle.provider/src/org/compiere/db/DB_Oracle.java

```diff
@@ -881,36 +656,119 @@ public Connection getDriverConnection (String dbUrl, String dbUid, String dbPwd)
         return DriverManager.getConnection (dbUrl, dbUid, dbPwd);
     }   //  getDriverConnection
 
-    /**
-     *  Close
-     */
-    public void close()
-    {
-        if (log.isLoggable(Level.CONFIG)) log.config(toString());
-        if (m_ds != null)
-        {
-        	try 
-			{
-				//wait 5 seconds if pool is still busy
-				if (m_ds.getNumBusyConnections() > 0)
-				{
-					Thread.sleep(5 * 1000);
-				}
-			} catch (Exception e)
-			{
+	private Properties getPoolProperties() {	
+		//check property file from home
+		File userPropertyFile = new File(getPoolPropertiesFile());
+		URL propertyFileURL = null;
+		
+		if (userPropertyFile.exists() && userPropertyFile.canRead())
+		{			
+			try {
+				propertyFileURL = userPropertyFile.toURI().toURL();
+			} catch (Exception e) {
 				e.printStackTrace();
 			}
-            try
-            {
-                m_ds.close();
-            }
-            catch (Exception e)
-            {
-                log.log(Level.SEVERE, "Could not close Data Source");
-            }
-        }
-        m_ds = null;
-    }   //  close
+		}
+			
+		if (propertyFileURL == null)
+		{
+			propertyFileURL = Ini.isClient()
+					? OracleBundleActivator.bundleContext.getBundle().getEntry("META-INF/pool/client.default.properties")
+							: OracleBundleActivator.bundleContext.getBundle().getEntry("META-INF/pool/server.default.properties");						

```

Comment:
Please remove isClient check and always load from server.default.properties

Reply:
Done! I've also removed the actual client properties file.

Reply:
seems all the fix for Ini.IsClient and client.properties is lost in the latest push